### ISSUE TITLE
[RFC] hr_employee_cost_history: add a comment field

### ISSUE THREAD

**Issue by sbidoul** [OCA/timesheet#696]
Thanks for the hr_employee_cost_history module.

Would it make sense to add a comment field on history line?
The use case is to document the reason for the change (salary raise, automatic indexation, etc).

If you agree, we can do a PR.

@rafaelbn @edlopen

**Comment by alexey-pelykh** [comment]
Should that be an HR contract or some paperwork attached?

**Comment by sbidoul** [comment]
> Should that be an HR contract or some paperwork attached?

Not necessarily. For instance in Belgium & Luxemburg there is an automatic salary indexation mechanism without any contractual change.

**Comment by edlopen** [comment]
It makes sense indeed. Ping me  in your PR to review it! Thank you!

**Comment by rafaelbn** [comment]
I'll test it too @sbidoul 👍🏼

**Comment by sbidoul** [comment]
Done in #735.

**Comment by rafaelbn** [comment]
@sbidoul you can close this one. PR is merging. Thank you!
